testsuite/MDAnalysisTests/test_api.py

```diff
@@ -49,3 +57,24 @@ def test_ResidueGroup():
 
 def test_SegmentGroup():
     assert mda.SegmentGroup is mda.core.groups.SegmentGroup
+
```

Comment:
Please don't forget to check the pep8speak comments!

(from a quick look, needs an extra empty line here and at line 69, and line 79 is too long).